javaparser-symbol-solver-core/src/main/java/com/github/javaparser/symbolsolver/javassistmodel/JavassistUtils.java

```diff
@@ -176,18 +177,26 @@ static String internalNameToCanonicalName(String typeName) {
     }
 
     private static ResolvedType objectTypeArgumentToType(SignatureAttribute.ObjectType typeArgument, TypeSolver typeSolver, ResolvedTypeParametrizable typeParametrizable) {
-        String typeName = typeArgument.jvmTypeName();
-        Optional<ResolvedType> type = getGenericParameterByName(typeName, typeParametrizable);
+        if(typeArgument instanceof SignatureAttribute.ArrayType){
```

Comment:
COuld we reuse https://github.com/javaparser/javaparser/blob/e201ce166190f2566633e7e285fde1f884888e0c/javaparser-symbol-solver-core/src/main/java/com/github/javaparser/symbolsolver/javassistmodel/JavassistUtils.java#L132 ?

Reply:
Yes, It looks like we could. Thank you for your remark, I will check it.